server.js

```diff
@@ -1,5 +1,5 @@
-const app = require('./app')
+const app = require('./src/app')
 
-app.listen('8080', function () {
+app.listen('8081', function () {
     console.log('App is listening on port 8080')
```

Comment:
The port has been changed to listen on 8081, and the console.log still shows a message of listening to port 8080.

Reply:
the port will be eventually moved to an .env key. so its not really an issue

Reply:
Ok great.